Notes/Development notes/260622 hyper vs whatsapp

```diff
@@ -0,0 +1,14 @@
+https://dev.to/shareef/make-your-terminal-beautiful-hyper-3jbm#https://git-scm.com/download/win
+
+hyper is something i used on my web course a while ago, it is built ontop of command prompt terminal i assume
```

Comment:
we can comment on lines of code here woody

Reply:
this doesnt effect the code is just notes while the pr is on. 
usually things like "hey this is nice" "what happens if"  "this should be a const not a let" ...